the-incredible-bulk/src/main/java/gov/va/api/health/bulkfhir/service/status/StatusRepository.java

```diff
@@ -19,6 +19,9 @@
   @Query("select s from StatusEntity s where s.buildCompleteEpoch = 0 and s.buildStartEpoch > 0")
   List<StatusEntity> findByStatusInProgress();
 
+  @Query("select s from StatusEntity s where s.buildCompleteEpoch = 0 and s.buildStartEpoch = 0")
```

Comment:
this could have also sorted and returned just a single entity.

Reply:
which would have been the cheapest solution, vs. load _all_ entities and sorting + `.get(0)` in the app.